homeassistant/components/lutron_caseta.py

```diff
@@ -0,0 +1,109 @@
+"""
+Component for interacting with a Lutron Caseta system.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/lutron_caseta/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (CONF_HOST,
+                                 CONF_USERNAME,
+                                 CONF_PASSWORD)
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['https://github.com/gurumitts/'
+                'pylutron-caseta/archive/v0.2.0.zip#'
+                'pylutron-caseta==v0.2.0', 'paramiko==2.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+LUTRON_CASETA_SMARTBRIDGE = 'lutron_smartbridge'
+LUTRON_CASETA_DEVICES = 'lutron_devices'
```

Comment:
Remove this. You can send the device id over the discovery and create the device on platform.